test/unit/action/constructor.test.js

```diff
@@ -1,18 +0,0 @@
-import Action from '../../../src/action'
-import Microcosm from '../../../src/microcosm'
-
-describe('Action constructor', function() {
-  it('an action payload is undefined by default', function() {
-    const action = new Action('test').resolve()
-
-    expect(action.payload).toBe(undefined)
-  })
-
-  it('an action can be set to a specific status', function() {
-    let repo = new Microcosm()
-
-    let action = repo.append('test', 'resolve')
-
-    expect(action.is('resolve')).toBe(true)
-  })
-})
```

Comment:
Removed. These are covered by other test suites and dig too much into internals.